apps/react-18-tests-v9/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/react--legacy"],
+  "root": true,
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
is this ok ? this not properly catch strict mode compliance which we should adhere to. not a blocker for this PR but going forward we should fix this.

Reply:
this was a copy over from whatever package I based this on. I'll try to remove.